docs/tutorials/github-keys.md

```diff
@@ -0,0 +1,68 @@
+---
+id: github-keys
+title: GitHub Keys
+sidebar_label: GitHub Keys 
+---
+
+## Why we need GitHub Keys?
+
+Let's first defined GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authetication.  Each user of the system has to generate a key. 
+
+The `SSH` is also known as Secure Shell or Secure Socket Shell [More About SSH in GitHub](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-ssh). SSH keys are used to authenticate secure connections they are generate through a public key cryptographic algorithm, the most common being RSA or DSA. This is a one-way formula that ensures the public key can be derived from the private key but the private key cannot be derived from the public key.
+
+
+## How to generate a new SSH key?
+
+This guide is for Windows users , more information for Linux and Mac [on this site](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+
+Windows environments do not have a standard default unix shell. External shell programs will need to be installed for to have a complete keygen experience. The most straight forward option is to utilize Git Bash. 
+
+- Open Git Bash
+
+- Paste the text below, substitute `"your_email@example.com"` with your GitHub email address.
+```shell
+$ ssh-keygen -t ed25519 -C "your_email@example.com"
+```
+---
+>**NOTE :** 
+If you are using a legacy system that doesn't support the `ed25519` algorithm, use:
+
+```shell
+$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+```
+---
+
+- It will ask for a file to save the key , just press enter. This accepts the default file location. 
```

Comment:
In this phrase, "It will ask for a file to save the key ," there is an extra empty space, before the comma mark.